ckan/logic/__init__.py

```diff
@@ -644,6 +644,14 @@ def wrapper(context, data_dict):
     return wrapper
 
 
+def chained_auth(func):
```

Comment:
I think we should rename this to `chained_auth_function` to distinguish from user authentication and because `IActions` -> `chained_action`; `IAuthFunctions` -> `chained_auth_function`

Reply:
Yep, good idea, thanks!